lib/fs.js

```diff
@@ -766,6 +774,14 @@ function mkdirSync(path, options) {
   handleErrorFromBinding(ctx);
 }
 
+Object.defineProperty(mkdirSync, features, {
```

Comment:
should we also add this property on the `fs.promises` implementation?

Reply:
I think that once we have consensus on this PR we should make a tracking issue and potentially get a bunch of different APIs properly setup with this

Reply:
oh, I was more specifically thinking we should have the symbol on the `fs.promises.mkdir` function.